webpack/assets/javascripts/react_app/components/charts/StatisticsChartsList.js

```diff
@@ -1,111 +1,49 @@
-import React, {PropTypes} from 'react';
-import helpers from '../../common/helpers';
+import React, { PropTypes } from 'react';
 import chartService from '../../../services/statisticsChartService';
 import ChartBox from './ChartBox';
-import { STATUS } from '../../constants';
-import StatisticsStore from '../../stores/StatisticsStore';
-import StatisticsChartActions from '../../actions/StatisticsChartActions';
 import './StatisticsChartsListStyles.css';
+import { connect } from 'react-redux';
+import * as StatisticsChartActions from '../../redux/actions/statistics';
+import { STATUS } from '../../constants';
 
-class StatisticsChartsList extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {charts: this.stateSetup(this.props.data)};
-
-    helpers.bindMethods(this, [
-      'onChange',
-      'onError']
-    );
+const getStatusFromChart = (chart) => {
```

Comment:
i think wrapper components are great, it's a very common pattern in react and functional programming, in common usecases.
i feel that there are cases where it's not required, and adds a level of complexity to the code.
if we have common functionality that spans several components, it's worth considering making a wrapper (if normal solutions such as css dont work out)...